packages/cletus/src/operations/dba.tsx

```diff
@@ -604,6 +604,164 @@ export const data_index = operationOf<
   },
 });
 
+export const data_import = operationOf<
+  { name: string; glob: string; chunkSize?: number; overlap?: number },
+  { imported: number; updated: number; skipped: number; libraryKnowledgeUpdated: boolean }
+>({
+  mode: 'create',
+  status: ({ name, glob }) => `Importing ${name} from ${glob}`,
+  analyze: async ({ name, glob }, { config, cwd }) => {
+    const type = getType(config, name);
+    const { glob: globLib } = await import('glob');
+    const files = await globLib(glob, { cwd });
+    
+    return {
```

Comment:
Follow patterns in clerk.tsx, search files and only import data from readable non-images (text, excel, doc, PDF, etc)